lib/ansible_tower_client/base_models/job_template.rb

```diff
@@ -25,5 +37,21 @@ def extra_vars_hash
     def override_raw_attributes
       { :credential => :credential_id, :inventory => :inventory_id, :project => :project_id }
     end
+
+    private
+
+    def check_ignorable_options(options)
+      ignored_options = IGNORABLE_OPTIONS.select do |option, checkmark|
+        !try(checkmark) && options.slice(option.to_sym, option.to_s).any?(&:present?)
+      end.keys
+
+      return if ignored_options.empty?
+
+      ignored_options.map! { |option| ":#{option}" }
```

Comment:
I think you are just trying to get a printable version of the symbol here, in which case you can just use `.inspect`...

```ruby
ignored_options.map!(&:inspect)
```

Reply:
done